packages/scene-composer/src/utils/sceneResourceUtils.ts

```diff
@@ -36,6 +36,8 @@ export const getSceneResourceDefaultValue = (type: SceneResourceType): string =>
       return IotTwinMakerNumberNamespace;
     case SceneResourceType.Opacity:
       return '1';
+    default:
```

Comment:
better to add the case for animation instead of adding default because for new cases it's more likely to have other kinds of values